packages/cli/src/lib/typeDistProject.ts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { findPaths } from '@backstage/cli-common';
+import { PackageRole } from '@backstage/cli-node';
+import { resolve as resolvePath } from 'path';
+import { Project, SourceFile, SyntaxKind, ts, Type } from 'ts-morph';
+
+export const createTypeDistProject = async () => {
+  const workspaceRoot = findPaths(process.cwd()).targetRoot;
```

Comment:
There's already `paths` in the CLI itself where you get this directly, iirc `lib/paths`